components/availability/src/init.spec.js

```diff
@@ -755,6 +712,48 @@ describe("Booking time slots", () => {
   });
 
   describe("change colours", () => {
+    const { availability, bookingOptions } = buildMockDataObjects([
+      {
+        emails: ["person@name.com"],
+        start_time: frozenDateTime().setHours(0, 0, 0, 0),
+        end_time: frozenDateTime().setHours(3, 0, 0, 0),
+      },
+      {
+        emails: ["person@name.com"],
+        start_time: frozenDateTime().setHours(6, 0, 0, 0),
+        end_time: frozenDateTime().setHours(9, 0, 0, 0),
+      },
+      {
+        emails: ["person@name.com"],
+        start_time: frozenDateTime().setHours(15, 0, 0, 0),
+        end_time: frozenDateTime().setHours(24, 0, 0, 0),
+      },
+      {
+        emails: ["thelonious@nylas.com"],
+        start_time: frozenDateTime().setHours(0, 0, 0, 0),
+        end_time: frozenDateTime().setHours(4, 0, 0, 0),
+      },
+      {
+        emails: ["thelonious@nylas.com"],
+        start_time: frozenDateTime().setHours(11, 0, 0, 0),
+        end_time: frozenDateTime().setHours(24, 0, 0, 0),
+      },
+      {
+        emails: ["booker@nylas.com"],
+        start_time: frozenDateTime().setHours(5, 15, 0, 0),
+        end_time: frozenDateTime().setHours(16, 0, 0, 0),
+      },
+    ]);
+
+    beforeEach(() => {
+      cy.get("@testComponent").then((element) => {
+        const component = element[0];
+
+        component.availability = availability;
+        component.booking_options = bookingOptions;
```

Comment:
@ozsivanov this is modifying existing tests - but does that not mean we're no longer testing data coming from the manifest with these changes? Shouldn't we have two sets of tests that load availability (a) manifest (b) prop?

Reply:
Yeah that was my initial plan, but it turns out that we have a mix of tests that use passed data and API mocks. I just updated the tests that were using `component.calendars = mockData` to make sure all of the tests were still passing exactly how they were before. I agree that we should have 2 sets of tests ideally, but this PR was getting pretty big already so I decided it'd be better for that to happen in the future.